steamtinkerlaunch

```diff
@@ -25776,8 +25776,9 @@ function checkSteamDeckDependencies {
 		local DEPFILENAME="$2"
 		local DEPDIR="$3"
 		local REPOURL="$4"
+		local CHECKCMD="$($DEPCMD &> /dev/null --version && echo "OK" || echo "NOK")"
```

Comment:
ShellCheck gives this warning:

> `SC2155 (warning): Declare and assign separately to avoid masking return values.`

ShellCheck gives a warning if you try to do this because it can potentially miss error codes, see more on the [SC2155 wiki page](https://www.shellcheck.net/wiki/SC2155).

I believe I ran into this a while back as well (honestly not sure why I went with locals for the Steam Deck stuff...) and just defined it separately.

One more universal solution. might be to break this out into a separate function (although note the ShellCheck syntax comments. But if you want to ignore this and justify why, we can add a ShellCheck ignore in a worst case (there are a couple of cases of this around the code, but I keep it to a minimum).